userspace/engine/rule_loader_collector.cpp

```diff
@@ -185,7 +191,7 @@ void rule_loader::collector::append(configuration& cfg, list_info& info)
 {
 	auto prev = m_list_infos.at(info.name);
 	THROW(!prev,
-	       "List has 'append' key but no list by that name already exists",
+	       "List has 'append' key or an append override but no list by that name already exists",
```

Comment:
Note for the reviewer: this string change is making our testing pipelines fail. No big deal, once we pick the right string I can update the pipelines.

Reply:
I have updated the code to revert the error messages, I really want to be sure that our tests are happy before merging. Messages can be updated in a subsequent PR coordinated with testing.